packages/components/src/LegacyFooter/LegacyMobileFooter.tsx

```diff
@@ -175,15 +175,17 @@ const legacyMobileSocialLinks: MobileSocialLinkType = [
   },
 ];
 
-const LegacyMobileSocialLinks = () => {
+export const LegacyMobileSocialLinks = () => {
```

Comment:
Das dürfte so die Styles aus dem `StyledLegacyMobileSocialLinks ` eigentlich gar nicht bekommen, weil der `className` aus den Props nicht übergeben wird. Und den margin kannst du doch auch einfach über die Box übergeben.

Reply:
Genau das wollte ich erreichen 😁 Also das war der Plan, dass das nicht übergeben wird. Im Footer brauche ich das Margin, überall anders aber nicht zwingend

Reply:
Aber, soweit ich das sehen kann, dürfte es das in beiden Fällen nicht bekommen. Also auch nicht bei der gewrappten Komponente. Dafür müsste `className` aus den Props noch an die Box übergeben werden.

Reply:
Ich schau nochmal :)